src/sas/qtgui/Perspectives/Corfunc/CorfuncPerspective.py

```diff
@@ -735,22 +735,22 @@ def getState(self):
         :return: {name: value}
         """
         return {
-            'guinier_a': self.txtGuinierA.text(),
-            'guinier_b': self.txtGuinierB.text(),
-            'porod_k': self.txtPorodK.text(),
-            'porod_sigma': self.txtPorodSigma.text(),
-            'avg_core_thick': self.txtAvgCoreThick.text(),
-            'avg_inter_thick': self.txtAvgIntThick.text(),
-            'avg_hard_block_thick': self.txtAvgHardBlock.text(),
-            'avg_soft_block_thick': self.txtAvgSoftBlock.text(),
+            'guinier_a': self.textGuinierA.text(),
+            'guinier_b': self.textGuinierB.text(),
+            'porod_k': self.textPorodK.text(),
+            'porod_sigma': self.textPorodSigma.text(),
+            'avg_core_thick': self.textAvgCoreThick.text(),
+            'avg_inter_thick': self.textAvgIntThick.text(),
+            'avg_hard_block_thick': self.textAvgHardBlock.text(),
+            'avg_soft_block_thick': self.textAvgSoftBlock.text(),
             'local_crystalinity': self.txtLocalCrystal.text(),
-            'polydispersity': self.txtPolyRyan.text(),
-            'polydispersity_stribeck': self.txtPolyStribeck.text(),
+            'polydispersity': self.textPolyRyan.text(),
+            'polydispersity_stribeck': self.textPolyStribeck.text(),
             'long_period': self.txtLongPeriod.text(),
             'lower_q_max': self.txtLowerQMax.text(),
             'upper_q_min': self.txtUpperQMin.text(),
             'upper_q_max': self.txtUpperQMax.text(),
```

Comment:
Hi Jeff, thanks for the comments. Of course the naming should be consistent, but i did not write a convention for naming down before and therefore mixed throughout the process. I will look the changes through and improve the things you suggested that have not been done yet.

The reason why i have not been editing all of the txt*, combobox* and checkbox* variable names is because in some cases the naming already was function sensitive and i did not want to mess up something in the logic in case i did not catch a variable that was renamed.